custom/import_xml.php

```diff
@@ -190,24 +193,29 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Import Patient Demographics','e'); ?></title>
+<?php Header::setupHeader(); ?>
+<title><?php xlt('Import Patient Demographics'); ?></title>
 </head>
 <body class="body_top" onload="javascript:document.forms[0].form_import_data.focus()">
-
-<p><?php xl('Paste the data to import into the text area below:','e'); ?></p>
-
-<center>
-<form method='post' action="import_xml.php">
-
-<textarea name='form_import_data' rows='10' cols='50' style='width:95%'></textarea>
-
-<p>
-<input type='submit' name='form_import' value=<?php xl('Import Patient','e','\'','\''); ?> /> &nbsp;
-<input type='button' value=<?php xl('Cancel','e','\'','\''); ?> onclick='window.close()' /></p>
+<form method='post' action="import_xml.php"> 
```

Comment:
In these legacy script, sometimes the form is missing the session restore stuff (this prevents session variables from leaking into other openemr instances in the web browser:
``````
<form method='post' action="import_xml.php" onsubmit="return top.restoreSession()">
``````
See here for details on this:
http://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows